notebooks/example.ipynb

```diff
@@ -45,7 +45,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 3,
```

Comment:
Even if you use the notebook for testing, I would avoid updating it unless you're making a change to the example, or if the output has changed and you want to show the updated output in the notebook preview. So in this case I would revert the changes to this file.